[2025-03-31T20:50:42.263-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-31T20:50:42.268-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_new_dag.load_and_clean_data scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-03-31T20:50:42.270-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_new_dag.load_and_clean_data scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-03-31T20:50:42.270-0300] {taskinstance.py:2303} INFO - Starting attempt 2 of 3
[2025-03-31T20:50:42.274-0300] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_and_clean_data> on 2025-02-01 00:00:00+00:00
[2025-03-31T20:50:42.277-0300] {standard_task_runner.py:63} INFO - Started process 10375 to run task
[2025-03-31T20:50:42.279-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'test_new_dag', 'load_and_clean_data', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/test_new_dag.py', '--cfg-path', '/var/folders/pw/ybzcyby54d787c124gv_5tjw0000gn/T/tmpg1u8oitu']
[2025-03-31T20:50:42.280-0300] {standard_task_runner.py:91} INFO - Job 6: Subtask load_and_clean_data
[2025-03-31T20:50:42.294-0300] {task_command.py:426} INFO - Running <TaskInstance: test_new_dag.load_and_clean_data scheduled__2025-02-01T00:00:00+00:00 [running]> on host ordenador-portatil-de-user-2.local
[2025-03-31T20:50:42.331-0300] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_new_dag' AIRFLOW_CTX_TASK_ID='load_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-03-31T20:50:42.331-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-31T20:50:42.472-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-31T20:50:42.472-0300] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/user/Documents/repo test eng/airflow_data/dags/test_new_dag.py", line 26, in load_and_clean
    from etl.data_loader import create_spark_session, load_all_tables
  File "/Users/user/Documents/repo test eng/airflow_data/dags/etl/data_loader.py", line 2, in <module>
    from config.settings import DB_CONFIG, JAR_PATH, TABLES
ModuleNotFoundError: No module named 'config'
[2025-03-31T20:50:42.475-0300] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=test_new_dag, task_id=load_and_clean_data, execution_date=20250201T000000, start_date=20250331T235042, end_date=20250331T235042
[2025-03-31T20:50:42.478-0300] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task load_and_clean_data (No module named 'config'; 10375)
[2025-03-31T20:50:42.512-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-03-31T20:50:42.520-0300] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-31T20:50:42.520-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
